### PR TITLE
added test for is_full_screen property

### PR DESCRIPTION
**Pull request by nileshgokhalepune**
Hi There, I am a new  python developer and a first time contributor to open source project. 
I recently started with toga and wanted to start contributing. So I followed the instructions on the toga contribute page and added a test. I am not sure if its worth adding a test for a property. The reason I added this test is to get that property in coverage .
I have executed all tests and confirm nothing else is broken. 

One more line covered under code coverage.
Tests whether the current_window is full screen. The test requires you to set the main window in full screen.

## PR Checklist:

- [ ] All new features have been tested
- [ ] All new features have been documented
- [x] I have read the **CONTRIBUTING.md** file
- [x] I will abide by the code of conduct

### CODE REVIEW

File: src/core/tests/test_app.py [hunk 1]
```diff
@@ -64,6 +64,10 @@ def test_app_exit(self):
         self.app.exit()
 
         self.assertActionPerformed(self.app, 'exit')
+
```
Comment (brutusthebee) [inline src/core/tests/test_app.py]: At column 1: (W293) blank line contains whitespace

File: src/core/tests/test_app.py [hunk 2]
```diff
@@ -64,6 +64,10 @@ def test_app_exit(self):
         self.app.exit()
 
         self.assertActionPerformed(self.app, 'exit')
+    
+    def test_app_is_full_screen(self):
+        self.app.set_full_screen(self.app.main_window)
+        self.assertEqual(self.app.is_full_screen, True)
```
Comment (brutusthebee) [inline src/core/tests/test_app.py]: At column 56: (W291) trailing whitespace

File: src/core/tests/test_app.py (hunk 1 above)
Comment (nileshgokhalepune) [reply src/core/tests/test_app.py]: White space removed.

File: src/core/tests/test_app.py (hunk 2 above)
Comment (nileshgokhalepune) [reply src/core/tests/test_app.py]: Trailing whitespace removed
```suggestion
        self.assertEqual(self.app.is_full_screen, True)    
```

**Comment by phildini** [comment]
Hi there! It looks like this PR might be dead, so we're closing it for now. Feel free to re-open it if you'd like to continue. 😄

**Pull request closed without merging**